pkg/sources/github/github.go

```diff
@@ -342,6 +343,12 @@ func checkGitHubConnection(ctx context.Context, client *github.Client) error {
 }
 
 func (s *Source) visibilityOf(ctx context.Context, repoURL string) (visibility source_metadatapb.Visibility) {
+	// It isn't possible to get the visibility of a wiki.
+	// We must use the visibility of the corresponding repository.
+	if strings.HasSuffix(repoURL, ".wiki.git") {
+		repoURL = strings.TrimSuffix(repoURL, ".wiki.git") + ".git"
```

Comment:
Does this happen under "normal" usage? As far as I can tell, this logic will only be executed if someone ties to scan a wiki repo directly - it won't run for wikis picked up as part of regular repo scans. Do I have that right?

Reply:
I believe it would get triggered in the following places via `github.cloneAndScanRepo` > `git.ScanRepo` > `git.ScanCommits`.

https://github.com/trufflesecurity/trufflehog/blob/28212c9a82221b1947674d5b5557486a7077d0c5/pkg/sources/git/git.go#L508-L510

https://github.com/trufflesecurity/trufflehog/blob/28212c9a82221b1947674d5b5557486a7077d0c5/pkg/sources/git/git.go#L529